boa/contracts/base_evm_contract.py

```diff
@@ -45,6 +45,12 @@ class StackTrace(list):
     def __str__(self):
         return "\n\n".join(str(x) for x in self)
 
+    @property
+    def dev_reason(self) -> str | None:
+        if isinstance(self.last_frame, str):
```

Comment:
oof, when can this happen?

Reply:
a couple lines later: https://github.com/DanielSchiavini/titanoboa/blob/18ab5b64dfe6ef6e41fc708fea9263c52e54a461/boa/contracts/base_evm_contract.py#L61